src/components/Accordion/Accordion.stories.tsx

```diff
@@ -33,7 +33,7 @@ export default {
     component: AccordionComponent,
 } as Meta<AccordionProps>;
 
-export const WithDifferentAccordionItems: Story<AccordionProps> = () => {
+export const WithDifferentAccordionItems: StoryFn<AccordionProps> = () => {
```

Comment:
Controls are looking funky, stuck in dark mode 🌚 
![Screenshot 2022-11-22 at 10 38 11](https://user-images.githubusercontent.com/35405438/203279857-c559f3e8-1d43-4843-8c69-f52f789055cf.png)

Reply:
Fixed